src/main/webapp/script.js

```diff
@@ -23,20 +23,37 @@ function getComments() {
       // Display the maximum number of comments
       for (let i = 0; i < data.length; i++) {
           // Format i comments in a list and display to the webpage.
-          if (data[i] != "") {
-              comments.appendChild(createListElement(data[i]));
-            }
+          if(i%3 == 0)
```

Comment:
This calculation seems like it'll be finicky in the future.

Formatting is off/not consistent here.